File: PurchasingAPI/Models/PurchaseViewModel.cs
namespace PurchasingAPI.Models;

public class PurchaseViewModel
{
    public PurchaseItem? PurchaseItem { get; set; }
    public PurchaseBilling? PurchaseBilling { get; set; }
    public string? FarmerName { get; set; }
    public string? VarietyName { get; set; }
}

File: UI/UIInterface/src/app/Models/purchaseviewmodel.ts
import { PurchaseItem } from "./purchase-item";
import { Purchasebilling } from "./purchasebilling";

export class Purchaseviewmodel {
  constructor(
    public farmerName: string,
    public purchaseBilling: Purchasebilling,
    public purchaseItem: PurchaseItem,
    public varietyName: string
  ){}
}